### DISCUSSION TITLE
Erigon crashes on stopping

### DISCUSSION THREAD

**Discussion by svetaU** [erigontech/erigon#11648] [General]
Hi! My erigon does not exit cleanly (sometimes). Not sure if it is a big issue. I have seen people reporting this behavior. Just in case attaching the log from stopping the service. It is 2.59
[log_for_github.txt](https://github.com/user-attachments/files/16640716/log_for_github.txt)

**Comment by AskAlexSharov** [comment]
start erigon with this env flags:
`GOGC=50 GOMEMLIMIT=20GiB ./build/bin/erigon --batchSize=128m`
